templates/ceph-storage.j2.j2

```diff
@@ -0,0 +1,35 @@
+---
+network_config:
+- type: interface
+  name: {{ ctlplane_interface }}
+  use_dhcp: false
+  {% raw -%}
+  dns_servers: {{ ctlplane_dns_nameservers }}
+  domain: {{ dns_search_domains }}
+  addresses:
+  - ip_netmask: {{ ctlplane_ip }}/{{ ctlplane_subnet_cidr }}
+  routes:
+  - ip_netmask: 0.0.0.0/0
+    next_hop: {{ ctlplane_gateway_ip }}
+    default: true
+{% endraw %}
+
+- type: ovs_bridge
+  name: br-isolated
+  use_dhcp: false
+  members:
+  - type: interface
+    name: {{ isolated_interface }}
+    primary: true
+  - type: vlan
+    vlan_id: {{ storage_network_vlan_id }}
```

Comment:
I think we should use ovs bridges only for tenant and external networks, as having other networks on ovs bridges is not recommended.